lib/afip_bill/views/shared/_factura_header.html.erb

```diff
@@ -184,8 +184,8 @@
   <div style="left:232.44px;top:169.29px" class="cls_005"><span class="cls_005">Hasta:</span><span class="cls_007"><%= Date.parse(afip_bill["fch_serv_hasta"]).strftime("%d/%m/%Y") %></span></div>
   <div style="left:362.96px;top:169.29px" class="cls_005"><span class="cls_005">Fecha de Vto. para el pago:</span></div>
   <div style="left:495.00px;top:169.29px" class="cls_007"><span class="cls_007"><%= Date.parse(afip_bill["fch_serv_hasta"]).strftime("%d/%m/%Y") %></span></div>
-  <div style="left:21.00px;top:189.59px" class="cls_002"><span class="cls_002">CUIT:</span></div>
-  <div style="left:52.00px;top:189.59px" class="cls_006"><span class="cls_006"><%= afip_bill["doc_num"] %></span></div>
+  <div style="left:21.00px;top:189.59px" class="cls_002"><span class="cls_002">CUIT/DNI:</span></div>
+  <div style="left:65.00px;top:189.59px" class="cls_006"><span class="cls_006"><%= afip_bill["doc_num"] %></span></div>
```

Comment:
@tcordeu Interesting! Couldn't you determine the label based on the value of `afip_bill["doc_num"]`? 

If it is a CUIT/CUIL value, what does that value look like? If it is a DNI what does it look like?